pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -53,6 +53,18 @@ type Endpoint struct {
 	// If true, certs verification will be skipped and HTTP (non-TLS)
 	// connections will be allowed.
 	Insecure bool `toml:"insecure,omitempty"`
+	// If true, the mirror will only be used for digest pulls. Pulling images by
+	// tag can potentially yield different images, depending on which endpoint
+	// we pull from.  Restricting mirrors to pulls by digest avoids that issue.
+	// This can only be set in a registry's Mirror field, not in the registry's primary Endpoint.
+	// If mirror-by-digest-only is set to true for the primary registry, this per-mirror setting is ignored.
+	DigestOnly bool `toml:"digest-only,omitempty"`
+	// If true, the mirror will only be used for tag pulls. Pulling images by
+	// tag can potentially yield different images, depending on which endpoint
+	// we pull from.  Restricting mirrors to pulls by digest avoids that issue.
```

Comment:
The `Pulling … issue` sentences are relevant as motivation for `DigestOnly`, and don’t seem that helpful for `TagOnly`.

I suppose that should be replaced by an explanation for why anyone would ever want `TagOnly` (= to have a more-up-to-date mirror that is less likely to be out of sync if tags move, but which is more expensive, so it should not be unnecessarily used for digest references).

… but it’s a bit unexpected that all this usage/rationale is in code comments, but not available in the end-user-facing `registries.conf.5.md`. If anything, I’d expect it the other way around.